org_chart.js

```diff
@@ -1,73 +1,138 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
-}
-
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
-  }
-]
-
-// assign each a manager
-let [alan, grace, donald] = new_hires
 
-alan.manager = elon
-grace.manager = elon
-donald.manager = elon
+function Employee(name, email, position, project){
+  this.name = name
+  this.email = email
+  this.position = position
+  this.skills = []
+  this.description = `${name} is a ${position} working on ${project}. He can be reached at ${email}`
+}
 
-// send to training
-if (!alan.skills) {
-  alan.skills = []
+Employee.prototype.set_manager = function(manager){
+  this.manager = manager
 }
-alan.skills.push("Mathematics")
 
-if (!grace.skills) {
-  grace.skills = []
+Employee.prototype.set_project = function(project){
+  this.project = project
 }
```

Comment:
It doesn't look like you are using `set_project()` any where.  If you are not using it, then there is no point in writing it.